types/node/node-tests.ts

```diff
@@ -38,10 +38,10 @@ import { Buffer as ImportedBuffer, SlowBuffer as ImportedSlowBuffer } from "buff
 //////////////////////////////////////////////////////////
 /// Global Tests : https://nodejs.org/api/global.html  ///
 //////////////////////////////////////////////////////////
-namespace global_tests {
+{
     {
-        let x: NodeModule;
-        let y: NodeModule;
+        const x: NodeModule = <any> {};
+        const y: NodeModule = <any> {};
```

Comment:
Same here. [`as` is recommended instead.](https://basarat.gitbooks.io/typescript/docs/types/type-assertion.html)